### ISSUE TITLE
Setting plugin variables works only with absolute paths

### ISSUE THREAD

**Issue by Trashcleaner** [heavenshell/vim-pydocstring#82]
Hello.
If I want to set variables, e.g. for doq installation or my template files, I have to type full path like this.
`let g:pydocstring_doq_path = '/home/user/.local/bin/doq'`

Using the tilda for home does not work and :Pydocstring command complains that it cannot find doq.
`let g:pydocstring_doq_path = '~/.local/bin/doq  " doesn't work'`

Similar is true for template_path
```
let g:pydocstring_templates_dir = '~/.vim/template_files/pydocstring_google_style'  " works
let g:pydocstring_templates_path = '/home/user/.vim/template_files/pydocstring_google_style'  " doesn't work
```

I think the first way is more portable for sharing dotfiles. Do you agree? Is a problem somewhere else?

Thank you.

**Comment by heavenshell** [comment]
You can set like
``
let g:pydocstring_templates_dir = expand( '~/.vim/template_files/pydocstring_google_style'  " works

**Comment by heavenshell** [comment]
fixed at 2.2.0
